pkg/types/llm/config.go

```diff
@@ -31,37 +31,37 @@ type Config struct {
 	Aliases              map[string]string  // Aliases maps short model names to full model names
 	Retry                RetryConfig        // Retry configuration for API calls
 
-	// Provider-specific configurations
```

Comment:
the comments on the struct fields shouldn't be removed - they need to be reverted back